src/store/modules/facility/getters.ts

```diff
@@ -53,6 +53,11 @@ const getters: GetterTree <FacilityState, RootState> = {
   },
   getTelecomAndEmailAddress(state) {
     return state.current?.contactDetails
+  },
+  isArchivedFacilitiesScrollable(state) {
+    return (
+      state.archivedFacilities.list?.length > 0 && state.archivedFacilities.list?.length < state.archivedFacilities.total
```

Comment:
I think this condition will not work as in all cases the list length and total will be same